opentelemetry-sdk/tests/test_configurator.py

```diff
@@ -685,8 +647,15 @@ def test_metrics_init_exporter(self):
             "otlp-service",
         )
         reader = provider._sdk_config.metric_readers[0]
-        self.assertIsInstance(reader, DummyMetricReader)
-        self.assertIsInstance(reader.exporter, DummyOTLPMetricExporter)
+        self.assertIsInstance(reader, PeriodicExportingMetricReader)
+        if python_implementation() == "PyPy" or version_info.minor == 7:
```

Comment:
Why is the check needed?

Reply:
Python 3.7 and PyPy provide a list in `self.metric_reader_mock.call_args[0]`.